src/webui/www/private/preferences_content.html

```diff
@@ -817,6 +817,279 @@
     </fieldset>
 </div>
 
+<div id="AdvancedTab" class="PrefTab invisible">
+    <fieldset class="settings">
+        <legend>QBT_TR(qBittorrent Section)QBT_TR[CONTEXT=OptionsDialog]&nbsp;(<a href="https://github.com/qbittorrent/qBittorrent/wiki/Explanation-of-Options-in-qBittorrent#Advanced" target="_blank">QBT_TR(Open documentation)QBT_TR[CONTEXT=HttpServer]</a>)</legend>
```

Comment:
We should drop the word `Section` in the WebUI and GUI. I think its intention is clear enough without it.

Reply:
It could happen in a later PR I presume (if that is the consensus).